templates/web-deployment.yaml

```
@@ -41,14 +42,24 @@ spec:
             - name: http
               containerPort: 80
               protocol: TCP
+          {{- if .Values.web.livenessProbe.enabled }}
           livenessProbe:
             httpGet:
-              path: /
-              port: http
+              path: {{ .Values.web.livenessProbe.path | default "/" }}
+              port: {{ .Values.web.service.port | default "http" }}
+            initialDelaySeconds: {{ .Values.web.livenessProbe.initialDelaySeconds | default 10 }}
+            timeoutSeconds: {{ .Values.web.livenessProbe.timeoutSeconds | default 20 }}
+          {{- end }}
+          {{- if .Values.web.readinessProbe.enabled }}
           readinessProbe:
             httpGet:
-              path: /
-              port: http
+              path: {{ .Values.web.readinessProbe.path | default "/" }}
+              port: {{ .Values.web.service.port | default "http" }}
+            initialDelaySeconds: {{ .Values.web.readinessProbe.initialDelaySeconds | default 10 }}
```

```suggestion
            initialDelaySeconds: {{ .Values.web.readinessProbe.initialDelaySeconds | default 5 }}
```

This container is a static web app running in nginx which boots within few seconds so the delay can be shorter to have a quicker response and autoscaling.